src/Payments/PaymentMethodsRecord.res

```diff
@@ -526,9 +533,21 @@ let getRequiredFieldsFromJson = dict => {
   }
 }
 
-let dynamicFieldsEnabledPaymentMethods = ["crypto_currency", "debit", "credit", "blik"]
+let dynamicFieldsEnabledPaymentMethods = [
+  "crypto_currency",
+  "debit",
+  "credit",
+  "blik",
+  "google_pay",
```

Comment:
Why is Google Pay added in this list? Does GooglePay also require additional fields?